docs/go-storage/services/ftp.md

```diff
@@ -22,6 +22,7 @@ Init storager (see [this page](../operations/index.md#how-to-initialize-a-servic
 
 ```go
 import (
+    ps "github.com/beyondstorage/go-storage/v4/pairs"
```

Comment:
Sort the import, plz.

Reply:
Sorry, I didn't get it. Should I add a new line after this or move this line to the end.

Reply:
Since the example indicate the ftp `Storager` used as a third lab, we can group these three package together. They should be sorted by name, like:
```
_ "github.com/beyondstorage/go-service-ftp"
ps "github.com/beyondstorage/go-storage/v4/pairs"
"github.com/beyondstorage/go-storage/v4/services"
```

Reply:
I see. The same lib should be sort in alphabet order by name.

Reply:
Bingo!